fe1-web/model/objects/RollCall.ts

```diff
@@ -81,8 +106,8 @@ export class RollCall implements LaoEvent {
       description: rc.description,
       creation: new Timestamp(rc.creation),
       start: new Timestamp(rc.start),
```

Comment:
fromState: 
opened_at
closed_at
proposed_start
proposed_end

toState, add:
- start, end

Reply:
I implemented this. 
I left the enum as a number enum, the reason is that in the `fromState` method, we need to cast a string back to an enum, and apparently, typescript doesn't support string -> string enum, only number -> number enum.
If that is not good, then I could write a method that does the conversion.

`public static fromState(rc: RollCallState): RollCall {
    return new RollCall({
      id: new Hash(rc.id),
      idAlias: (rc.idAlias) ? new Hash(rc.idAlias) : undefined,
      name: rc.name,
      location: rc.location,
      description: rc.description,
      creation: new Timestamp(rc.creation),
      proposed_start: new Timestamp(rc.proposed_start),
      proposed_end: new Timestamp(rc.proposed_end),
      opened_at: new Timestamp(rc.opened_at),
      closed_at: new Timestamp(rc.closed_at),
      status: rc.status, // Here
      attendees: rc.attendees?.map((a) => new PublicKey(a)),
    });
  }`